metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -79,7 +79,7 @@ def cli():
 @click.pass_obj
```

Comment:
@savingoyal fwiw, if our build checks dont account for mocked argo/airflow/SF runs, it is almost a good idea for me/team to also check, if substitutions of pylint with ruff and linter terminologies/capabilities isn't introducing any regression/breaking changes failures here. Thoughts?